browser_wasm.go

```diff
@@ -54,7 +54,7 @@ func CreateWindow(_, _ int, title string, monitor *Monitor, share *Window) (*Win
 	canvas.Set("height", int(float64(height)*devicePixelRatio+0.5)) // Nearest non-negative int.
 
 	canvas.Get("style").Call("setProperty", "width", "100vw")
```

Comment:
Not needed, but maybe we should also use `dvw` for width (?)

```suggestion
	canvas.Get("style").Call("setProperty", "width", "100dvw")
```